switching/input/messages/R1.py

```diff
@@ -732,3 +748,124 @@ def detall_resultat(self):
             pass
         return ref
 
+
+class MinimumFieldsChecker(object):
+    def check_nif_cliente(self, r1):
+        return getattr(getattr(r1, 'client', None), 'codi_identificacio', None)
```

Comment:
Try to use [get_rec_attr](https://github.com/gisce/switching/blob/master/switching/helpers/funcions.py#L24)